cfpb/consumerfinance.gov
# Add notice and consent warning, server indicator banner

The primary purpose of this PR is to add the CFPB's "notice and consent" warning to comply with [NIST Special Publication 800-53 (Rev. 4) control AC-8](https://nvd.nist.gov/800-53/Rev4/control/AC-8).

More interestingly, I took the opportunity to reduce the customization of our custom login page template and bring it up to date with the latest in the current release of Wagtail. This includes removing our customized heading logic, which I suspect never actually clues anyone in to the fact that they're using Build. To compensate, though, I have implemented a little banner at the top of the left-hand menu (or all the way across the top on narrower viewports) that highlights what server you're on. 

---

## Additions

- Notice and consent warning added to login page
- Server indicator banner added to all pages


## Removals

- Some of the customization in our custom login template


## How to test this PR

1. Pull branch
1. `gulp scripts:admin`
1. Load up http://localhost:8000/admin/ and look at the login page. Log in to see the banner in its idea placement at the top of the sidebar.


## Screenshots

![Login screen with notice and consent warning](https://user-images.githubusercontent.com/1044670/100939401-36a5c500-34c4-11eb-803b-85c9793c9da1.png)


![Server banner on a wide viewport](https://user-images.githubusercontent.com/1044670/100939341-1f66d780-34c4-11eb-9a58-89d977683df8.png)

![Server banner on a narrow viewport](https://user-images.githubusercontent.com/1044670/100939879-0b6fa580-34c5-11eb-95b8-281ac7d72baa.png)


## Notes and todos

- [x] Confirm with Gerald that this text, which I copied from our phone lock screens, is acceptable.
- At the moment, the banner is always the same color, but it might be nice to have it be color-coded for our common environments.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
